proposals/0475-tuple-syntax.rst

```diff
@@ -63,18 +63,26 @@ variable in a bit of code. This allows us to e.g. refer to ``Tuple2`` and ``Tupl
 Proposed Change Specification
 -----------------------------
 
-1. Create a new module in ``base`` called ``GHC.Prelude``. This module exports definitions
-   peculiar to GHC that are safe to use in ordinary code.
-   This module is *not* intended as a replacement to ``Prelude``,
-   but instead a complement to it. This module is a counterpart to ``GHC.Exts``, which exports
+1. Create three new modules in ``ghc-experimental`` called ``Prelude.Experimental``,
+   ``Data.Tuple.Experimental`` and ``Data.Sum.Experimental``.
+   Since this new paradigm may evolve in response to users' experience, we want
+   to avoid committing to an API in ``base`` too early, and therefore choose the
+   new package as the home for the exports of the new entities.
+
+   ``Prelude.Experimental`` exports definitions peculiar to GHC that are safe to
+   use in ordinary code, but whose API may evolve rapidly over subsequent releases.
+   This module is *not* intended as a replacement to ``Prelude``, but instead a
+   complement to it. This module is a counterpart to ``GHC.Exts``, which exports
    many *unsafe* internals.
 
-   This proposal lists several definitions to be exported from ``GHC.Prelude``, but leaves other
-   definitions to be added in separate proposal(s).
+   This proposal lists several definitions to be exported from ``Prelude.Experimental``,
+   but leaves other definitions to be added in separate proposal(s).
+
+   All of the wired-in types must be declared in ``ghc-prim``, and re-exported from
+   ``ghc-experimental``.
 
-#. Export the following definitions from ``GHC.Tuple`` and ``GHC.Prelude``. (Implementation note:
-   These would probably end up in a new module ``GHC.Tuple.Prim``, due to the dependency from
-   the definitions below in ``GHC.Tuple`` on ``GHC.TypeLits``.) ::
+#. Replace the existing tuples in ``GHC.Tuple`` with the following definitions
+   and export them from ``Data.Tuple.Experimental`` and ``Prelude.Experimental``. ::
```

Comment:
I would not talk about replacing anything.  That's an implementation matter.  Perhaps just
```
Export the following type definitions from `Data.Tuple.Experimental`.